db_migrate/env.py

```diff
@@ -17,7 +17,7 @@
 
     sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__),
                                                  "..")))
-    from libcodechecker.server.run_db_model import Base
+    from libcodechecker.server.database.run_db_model import Base
```

Comment:
Do this modification also in the config_db_migrate:
https://github.com/Ericsson/codechecker/blob/50ba32b02b06b90c86eabf50554fe9bb45752f3a/config_db_migrate/env.py#L22

Reply:
done.